accounts/erd1qqqqqqqqqqqqqpgqwhfa0jpuuv8gk7qsqzgmuhq438zlhmpvpwasvqx7e7.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "ErdLaunchpad: Staking",
+  "description": "Staking smart contract of ErdLaunchpad",
```

Comment:
Please use the following format:
Application: Logic name

In your case would be:
ErdLaunchpad: Staking smart contract